kube/lib/holdings/config.libsonnet

```diff
@@ -0,0 +1,30 @@
+{
+  _config+:: {
+    holdings: {
+      mysql: {
+        port: 3306,
+        ip: '10.255.8.249',
```

Comment:
I don't love repeating this magic IP address across a bunch of public repos. Consul will save us from needing to do this imminently, though.

Reply:
@daaang @botimer  Care to respond to this?